back/spec/lib/participation_method/volunteering_spec.rb

```diff
@@ -63,4 +63,11 @@
       expect(participation_method.extra_fields_category_translation_key).to eq 'custom_forms.categories.extra.title'
     end
   end
+
+  its(:supports_publication?) { is_expected.to be false }
```

Comment:
Indeed they are not. I think we should use them, because the participation methods define which actions can be taken on an input. When we slowly move towards the UDM, the participation methods have to play a bigger role than they are playing to day. For instance, these methods can be used when validating Idea objects.